### ISSUE TITLE
Translate documentation

### ISSUE THREAD

**Issue by SamR1** [SamR1/FitTrackee#45]

**Comment by Thovi98** [comment]
I just discovered your app in my search for a self-hosted alternative to Strava. It looks promising, especially when I see that the development for the social features has already started! I'll try to install it when I have the time and skill.

I'm not a developer, but I can help translate some things (in French especially), like the documentation. Is there any software like Weblate or Transifex for this purpose? How can I help?

**Comment by Thovi98** [comment]
> If you want, I can get back to you when it's done.

Yes very nice thank you ! But take your time, I don’t have so many time right now. 
Thanks for your answer :)
